Makefile

```diff
@@ -257,6 +257,11 @@ install-govmomi:
 
 test: install-govmomi portlayerapi $(TEST_JOBS)
 
+
+focused-test:
+# test only those packages that have changes
+	infra/scripts/focused-test.sh $(REMOTE)
```

Comment:
It's not - if you call it in the following style it'll get passed through, otherwise it tries to determine a default in the script:
```
REMOTE=vmware/master make focused-test
```